src/AccordionItem.svelte

```diff
@@ -4,6 +4,7 @@
   export let expanded = false;
   export let disabled = false;
   export let buttonClass = "";
+  export let style = "default";
```

Comment:
Calling the class name "style" may be misleading. Also, it could be overridden by `$$restProps.style`.